alexthompson207/whats-cookin
# Feature/remove favorite recipes

## Is this a fix or a feature?
-feature

## What is the change?

- Favorite recipe cards will now be displayed with an X icon to indicate that it can be deleted
-  when user clicks on the X, the card will be removed from display and from their favorite recipe list


## What does it fix?
- N/A
## Where should the reviewer start?
- script.js: lines 44-56, 122-132, 210-212
- 
## How should this be tested?
- Run index.html to open up the application
- Click on a recipe and add a few favorites
- Go to the favorites area and click on the X
- The recipe should be removed

## Screenshots (if appropriate):

<img width="1414" alt="Screen Shot 2021-02-13 at 9 35 50 PM" src="https://user-images.githubusercontent.com/72578730/107868681-7b929400-6e43-11eb-96a6-c03d0006d05a.png">
